### PR TITLE
junit: allow multiple properties with same name

### LINKED ISSUE #1422: junit record_xml_property doesn't allow multiple records with same name
It might happen that test can be affected by two or more bugs.
I need to be able to track them all.

```
def test_record_with_same_name(record_xml_property):
    record_xml_property("bug", "X")
    record_xml_property("bug", "Y")
```

py,test 2.8.7

### PR DESCRIPTION
**Pull request by lukas-bednar**
It might happen that test can be affected by two or more bugs.
I need to be able to track them all.

Fix #1422

### CODE REVIEW

**Comment by RonnyPfannschmidt** [comment]
looks good to merge, is there a special reason why both, properties and the insert-order is needed?

i wonder if the changes section might be a better place in the changelog

**Comment by The-Compiler** [comment]
LGTM, but I'm not familiar with that part of the code and JUnit - so I'll leave this one to @RonnyPfannschmidt, @nicoddemus or anyone else who feels inclined :wink:

@dajose and @lukas-bednar - could you give this a look as well?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -98,7 +98,8 @@
   (auto/long/short/line/native/no), with `auto` being the default since v2.6.
   Thanks `@hackebrot`_ for the PR.
 
-*
+* Fix (`#1422`): junit record_xml_property doesn't allow multiple records
```
Comment (lukas-bednar) [inline CHANGELOG.rst]: ops, will append underscore

**Comment by nicoddemus** [comment]
LGTM as well! :smile: 

Once it passes on CI we can merge it.

**Pull request merged**